src/Microsoft.CodeQuality.Analyzers/UnitTests/Maintainability/RemoveUnusedLocalsTests.Fixer.cs

```diff
@@ -30,5 +30,527 @@ protected override CodeFixProvider GetCSharpCodeFixProvider()
+        }
+
+        [Fact]
+        public void UnusedLocal_BaseFunctionality_Basic()
+        {
+            var code = @"
+Public Class C
+    Function F() As Integer
+        Dim a As Integer = 0 ' inline comment also to be deleted. 
+        Dim b As Integer
+        b = 0
+        Return 0
+    End Function
+
+    Function G() As Integer
+        Dim a As Integer = 0
+        Return a
+    End Function
+End Class
+";
+            var fix = @"
+Public Class C
+    Function F() As Integer
+        Return 0
+    End Function
+
+    Function G() As Integer
+        Dim a As Integer = 0
+        Return a
+    End Function
+End Class
+";
+            VerifyBasicFix(code, fix, allowNewCompilerDiagnostics: true);
+            VerifyBasicFixAll(code, fix, allowNewCompilerDiagnostics: true);
+        }
+
+        [Fact]
+        public void UnusedLocal_ParametersAndMethodCalls_Basic()
+        {
+            var code = @"
+Public Class C
+    Function F() As Integer
+        Dim a As Integer = 0
+        Dim b As Integer = G(a)
+        Dim c As Integer
+        c = G(a)
+        Return 0
+    End Function
+
+    Function G(p As Integer) As Integer
+        Return 1
+    End Function
+End Class
+";
+          
+            VerifyBasicFix(code, code, allowNewCompilerDiagnostics: true);
+            VerifyBasicFixAll(code, code, allowNewCompilerDiagnostics: true);
+        }
+
+        [Fact]
+        public void UnusedLocal_JointDeclaration_RemoveFirst_Basic()
+        {
+            var code = @"
+Public Class C
+    Function F() As Integer
+        Dim a As Integer, b As Integer
+        Dim c = 0, d = 0
+        a = 0
+        b = 0
+        Return b + d
+    End Function
+End Class
+";
+            var fix = @"
+Public Class C
+    Function F() As Integer
+        Dim b As Integer
+        Dim d = 0
+        b = 0
+        Return b + d
+    End Function
+End Class
+";
+            VerifyBasicFix(code, fix, allowNewCompilerDiagnostics: true);
+            VerifyBasicFixAll(code, fix, allowNewCompilerDiagnostics: true);
+        }
+
+        [Fact]
+        public void UnusedLocal_JointDeclaration_RemoveSecond_Basic()
+        {
+            var code = @"
+Public Class C
+    Function F() As Integer
+        Dim a As Integer, b As Integer
+        Dim c = 0, d = 0
+        a = 0
+        b = 0
+        Return a + c
+    End Function
+End Class
+";
+            var fix = @"
+Public Class C
+    Function F() As Integer
+        Dim a As Integer
+        Dim c = 0
+        a = 0
+        Return a + c
+    End Function
+End Class
+";
+            VerifyBasicFix(code, fix, allowNewCompilerDiagnostics: true);
+            VerifyBasicFixAll(code, fix, allowNewCompilerDiagnostics: true);
+        }
+
+        [Fact]
+        public void UnusedLocal_JointDeclaration_RemoveBoth_Basic()
+        {
+            var code = @"
+Public Class C
+    Function F() As Integer
+        Dim a As Integer, b As Integer
```

Comment:
Can you add a test for: `Dim a, b As New Integer` for (a) Only one of them is unused and (b) Both of them are unused?